usecase/usecase-webapi-xwalk-tests/testscripts/BrowserState.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Usecase WebAPI
+ Scenario: Device & Hardware/BrowserState Test
+    When launch "usecase-webapi-xwalk-tests"
+     And I go to "/samples/BrowserState/index.html"
+     And I open wifi
```

Comment:
If the WIFI status is closed, open/close airplane mode can not connect to internet.